<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>691</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>301</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>字符串截取</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>291</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="Pushbutton_frontback">
             <property name="text">
              <string>front&amp;back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_leftright">
             <property name="text">
              <string>leftright</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_fristlast">
             <property name="text">
              <string>firstlast</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_section">
             <property name="text">
              <string>section*</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>301</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>存储相关</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_Nullempty">
             <property name="text">
              <string>isNULLorempty</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_resize">
             <property name="text">
              <string>resize</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_size">
             <property name="text">
              <string>sizeandlen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_fill">
             <property name="text">
              <string>fill</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>301</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>搜索与判断</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_index">
             <property name="text">
              <string>index</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_contains">
             <property name="text">
              <string>contains</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButton_endstrtswith">
             <property name="text">
              <string>endwith||startswith</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_count">
             <property name="text">
              <string>count</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>301</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>转换和修改</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="pushButton_trimmed">
             <property name="text">
              <string>trimmed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_inserts">
             <property name="text">
              <string>inserts</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="pushButton_remove">
             <property name="text">
              <string>remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_replace">
             <property name="text">
              <string>replace</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>331</width>
         <height>451</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>311</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>输入</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>69</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>str1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>69</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>str2</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_str1">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>C:\QT82625\xaom.md</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBox_str2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>87</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>\</string>
          </property>
         </item>
        </widget>
        <widget class="QSpinBox" name="spinBox_num">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>60</y>
           <width>44</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>311</width>
          <height>311</height>
         </rect>
        </property>
        <property name="title">
         <string>结果显示</string>
        </property>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>281</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>281</width>
           <height>241</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
